packages/react-native-renderer/src/ReactNativeComponent.js

```diff
@@ -149,7 +149,7 @@ class ReactNativeComponent<DefaultProps, Props, State> extends React.Component<
     }
 
     const viewConfig: ReactNativeBaseComponentViewConfig =
-      maybeInstance.viewConfig;
+      maybeInstance.viewConfig || maybeInstance.canonical.viewConfig;
```

Comment:
Why does this code need to work with Fabric instances? Do you mean we should pull it out and be shared between the two?